### PR TITLE
i18n(es): add deno guide

### PR DESCRIPTION
**Pull request by miguelfdezc**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Adds `deno.md` in spanish

### CODE REVIEW

**Comment by Yan-Thomas** [comment]
Thanks for this @miguelfdezc, I couldn't find you on our Discord server yet, so if you're interested in joining our i18n squad and have access to the ES translation thread where you can share your PRs and talk with the rest of the team, it's just a click away: https://astro.build/chat

File: src/pages/es/guides/deploy/deno.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+### Despliegue con Github Actions
+
+Si tu proyecto está almacenado en GitHub, el [sitio web de Deno Deploy](https://dash.deno.com/) te guiará en la configuración de Github Actions para desplegar tu sitio de Astro.
+
+1. Sube tu código a un repositorio público o privado de Github.
+
+1. Inicia sesión en [Deno Deploy](https://dash.deno.com/) con tu cuenta de Github y haz clic en [Nuevo Proyecto](https://dash.deno.com).
+
+1. Selecciona tu repositorio, la rama desde la que quieres desplegar, y selecciona el modo **Github Action**. (Tu sitio Astro requiere un paso de construcción, y no puede usar el modo Automático).
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:77]:
```suggestion
3. Selecciona tu repositorio, la rama desde la que quieres desplegar, y selecciona el modo **Github Action**. (Tu proyecto de Astro requiere un paso de construcción, y no puede usar el modo Automático).
```

File: src/pages/es/guides/deploy/deno.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+   
+1. En tu proyecto de Astro, crea un nuevo archivo en `.github/workflows/deploy.yml` y pega el YAML de abajo. Esto es similar al YAML dado por Deno Deploy, con los pasos adicionales necesarios para tu sitio de Astro.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:79]:
```suggestion
4. En tu proyecto de Astro, crea un nuevo archivo en `.github/workflows/deploy.yml` y pega el YAML a continuación. Esto es similar al YAML dado por Deno Deploy, con los pasos adicionales necesarios para tu proyecto de Astro.
```

File: src/pages/es/guides/deploy/deno.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+          - name: Subir a Deno Deploy
+            uses: denoland/deployctl@v1
+            with:
+              project: my-deno-project # TODO: sustituir por el nombre del proyecto Deno Deploy
+              entrypoint: server/entry.mjs
+              root: dist
+    ```
+
+1. Después de hacer commit de este archivo YAML, y de subir a Github en tu rama de despliegue configurada, ¡el despliegue debe comenzar automáticamente!
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:113]:
```suggestion
5. Después de hacer commit de este archivo YAML, y de subir a Github en tu rama de despliegue configurada, ¡el despliegue comenzará automáticamente!
```

File: src/pages/es/guides/deploy/deno.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+   Puedes seguir el progreso utilizando la pestaña "Actions" en la página de tu repositorio de Github, o en [Deno Deploy](https://dash.deno.com).
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:115]:
```suggestion
   Puedes seguir el progreso utilizando la pestaña "Actions" de tu repositorio de Github, o en [Deno Deploy](https://dash.deno.com).
```

File: src/pages/es/guides/deploy/deno.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+
+### Despliegue con la CLI
+
+1. Instala la [Deno Deploy CLI](https://deno.com/deploy/docs/deployctl).
+
+    ```bash
+    deno install --allow-read --allow-write --allow-env --allow-net --allow-run --no-check -r -f https://deno.land/x/deploy/deployctl.ts
+    ```
+
+1. Ejecuta el paso de construcción de tu proyecto de Astro.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:126]:
```suggestion
2. Ejecuta el paso de construcción de tu proyecto de Astro.
```

File: src/pages/es/guides/deploy/deno.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+    ```bash
+    npm run build
+    ```
+
+1. ¡Ejecuta `deployctl` para desplegar!
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:132]:
```suggestion
3. ¡Ejecuta `deployctl` para desplegar!
```

File: src/pages/es/guides/deploy/deno.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+   En el siguiente comando, sustituye `<ACCESS-TOKEN>` por tu [Token de Acceso Personal](https://dash.deno.com/user/access-tokens) y `<MY-DENO-PROJECT>` por el nombre de tu proyecto en Deno Deploy.
+
+    ```bash
+    DENO_DEPLOY_TOKEN=<ACCESS-TOKEN> deployctl deploy --project=<MY-DENO-PROJECT> --no-static --include=./dist ./dist/server/entry.mjs
+    ```
+    
+    Puedes seguir todos tus despliegues en [Deno Deploy](https://dash.deno.com).
+
+1. (Opcional) Para simplificar la construcción y el despliegue en un solo comando, añade un script `deploy-deno` en `package.json`.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:142]:
```suggestion
4. (Opcional) Para simplificar la construcción y el despliegue en un solo comando, añade un script `deploy-deno` en `package.json`.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Amazing job, a few comments to match our docs style 🇪🇸

File: src/pages/es/guides/deploy/deno.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+Para activar el SSR en tu proyecto de Astro y desplegarlo en Deno Deploy:
+
+1. Instala [el adaptador Deno](https://github.com/withastro/astro/tree/main/packages/integrations/deno) en las dependencias de tu proyecto.
+
+    ```bash
+      npm install @astrojs/deno
+    ```
+
+1. Update your `astro.config.mjs` project configuration file with the changes below.
```
[the remark this answers is not in the document]
Comment (fredoist) [reply src/pages/es/guides/deploy/deno.md:30]: Solo utilizar _"Actualiza"_ en lugar de _"Actualice"_ para coincidir con el resto del documento.

**Comment by agustinmulet** [comment]
@miguelfdezc Please mark the changes made as resolved :) 

LGTM! ✅

**Comment by Yan-Thomas** [comment]
Welcome to the docs' face pile @miguelfdezc! 🥳 
Thank you all for this amazing job 🙌

**Pull request merged**